### DISCUSSION TITLE
Dev mode - Dependencies of examples aren't updated durinb `pnpm dev`

### DISCUSSION THREAD

**Discussion by vitalyb-qwilt** [scalar/scalar#1838] [Q&A] [answered]
Hi,

I am trying to build and develop scalar locally. However, even when I am running in dev mode and opening `examples/react` if I do any change in `api-reference` I don't see any changes until I stop everything and run `pnpm build` again.

Is there a way to get these change quicker during development? Is there something like hot-reload in this project?

**Comment by hanspagel** [comment, marked as answer]
You can use the @scalar/api-reference examples on http://localhost:5050 to develop as a workaround. Does that work for you?

From the top of my head I don’t remember why we don’t have HMR for the react example, but it might be a limitation of our setup - or we just didn’t set this up yet.

**Comment by vitalyb-qwilt** [comment]
Oh that works great! Thanks!
